docs/source/models/cause.rst

```diff
@@ -337,14 +336,38 @@ Mortality Impacts
 Cause-Specific Mortality
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
+The cause-specific mortality rate is the mortality rate from a specified cause for a population. 
+The numerator is the number of deaths attributed to a specific cause. The denominator remains the 
+size of the population. In GBD, cause-specific estimates by country, year, age, sex.
+
+  For example, 112 measles deaths in country A during calendar year 2016. 1010,623 estimated 2016 mid-year 
+  population. 11.08 measles deaths per 100,000 population (person-years at risk) during calendar year 
```

Comment:
mid-year population * one year is how to go from the population count to an estimate of person-years.